src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,24 @@
+.clock {
+  margin: 25vh auto;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: $clockSize;
+  height: $clockSize;
+
+  border-radius: $circle;
+  background-color: #5b6264;
+  animation: shape 5s linear 0s infinite alternate;
+
+  &__middle {
+    position: absolute;
+    display: flex;
```

Comment:
These 2 properties aren't needed. You already centered it with parent element flex properties.  Code works fine without them.